tsconfig.json

```diff
@@ -9,6 +9,7 @@
 		"declaration": true
 	},
 	"files": [
+    "typings/main.d.ts",
```

Comment:
It's needed for `tsc` to compile with no warnings when running `npm run build_dist`.
Otherwise, you get:
```
node_modules/angular2/src/core/change_detection/parser/locals.d.ts(3,14): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/change_detection/parser/locals.d.ts(4,42): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/element_injector.d.ts(71,32): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/element_injector.d.ts(73,17): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/element_injector.d.ts(77,184): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/element_injector.d.ts(82,182): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/element_injector.d.ts(106,37): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/proto_view_factory.d.ts(28,146): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/view.d.ts(52,144): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/view.d.ts(76,79): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/view.d.ts(77,73): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/view.d.ts(95,31): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/view.d.ts(98,18): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/view.d.ts(101,24): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/view.d.ts(104,162): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/linker/view.d.ts(105,160): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/render/api.d.ts(317,74): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/core/zone/ng_zone.d.ts(2,37): error TS2304: Cannot find name 'Zone'.
node_modules/angular2/src/facade/collection.d.ts(1,25): error TS2304: Cannot find name 'MapConstructor'.
node_modules/angular2/src/facade/collection.d.ts(2,25): error TS2304: Cannot find name 'SetConstructor'.
node_modules/angular2/src/facade/collection.d.ts(4,27): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/facade/collection.d.ts(4,39): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/facade/collection.d.ts(7,9): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/facade/collection.d.ts(8,30): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/facade/collection.d.ts(11,43): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/facade/collection.d.ts(12,27): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/facade/collection.d.ts(14,23): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/facade/collection.d.ts(15,25): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/facade/collection.d.ts(94,41): error TS2304: Cannot find name 'Set'.
node_modules/angular2/src/facade/collection.d.ts(95,22): error TS2304: Cannot find name 'Set'.
node_modules/angular2/src/facade/collection.d.ts(96,25): error TS2304: Cannot find name 'Set'.
node_modules/angular2/src/facade/lang.d.ts(2,22): error TS2304: Cannot find name 'BrowserNodeGlobal'.
node_modules/angular2/src/facade/lang.d.ts(48,59): error TS2304: Cannot find name 'Map'.
node_modules/angular2/src/facade/promise.d.ts(1,10): error TS2304: Cannot find name 'Promise'.
node_modules/angular2/src/facade/promise.d.ts(3,14): error TS2304: Cannot find name 'Promise'.
node_modules/angular2/src/facade/promise.d.ts(8,32): error TS2304: Cannot find name 'Promise'.
node_modules/angular2/src/facade/promise.d.ts(9,38): error TS2304: Cannot find name 'Promise'.
node_modules/angular2/src/facade/promise.d.ts(10,35): error TS2304: Cannot find name 'Promise'.
node_modules/angular2/src/facade/promise.d.ts(10,93): error TS2304: Cannot find name 'Promise'.
node_modules/angular2/src/facade/promise.d.ts(11,34): error TS2304: Cannot find name 'Promise'.
node_modules/angular2/src/facade/promise.d.ts(12,32): error TS2304: Cannot find name 'Promise'.
node_modules/angular2/src/facade/promise.d.ts(12,149): error TS2304: Cannot find name 'Promise'.
node_modules/angular2/src/facade/promise.d.ts(13,43): error TS2304: Cannot find name 'Promise'.
node_modules/rxjs/CoreOperators.d.ts(22,67): error TS2304: Cannot find name 'Promise'.
node_modules/rxjs/CoreOperators.d.ts(72,67): error TS2304: Cannot find name 'Promise'.
node_modules/rxjs/CoreOperators.d.ts(77,31): error TS2304: Cannot find name 'PromiseConstructor'.
node_modules/rxjs/CoreOperators.d.ts(77,54): error TS2304: Cannot find name 'Promise'.
node_modules/rxjs/Observable.d.ts(65,67): error TS2304: Cannot find name 'PromiseConstructor'.
node_modules/rxjs/Observable.d.ts(65,88): error TS2304: Cannot find name 'Promise'.
node_modules/rxjs/Observable.d.ts(72,84): error TS2304: Cannot find name 'Promise'.
node_modules/rxjs/Observable.d.ts(77,38): error TS2304: Cannot find name 'Promise'.
node_modules/rxjs/Observable.d.ts(100,66): error TS2304: Cannot find name 'Promise'.
node_modules/rxjs/Observable.d.ts(154,66): error TS2304: Cannot find name 'Promise'.
node_modules/rxjs/Observable.d.ts(159,31): error TS2304: Cannot find name 'PromiseConstructor'.
node_modules/rxjs/Observable.d.ts(159,54): error TS2304: Cannot find name 'Promise'.
src/store.ts(42,25): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
src/store.ts(50,29): error TS2339: Property 'assign' does not exist on type 'ObjectConstructor'.
```